lib/cucumber/glue/proto_world.rb

```diff
@@ -111,8 +112,13 @@ def log(*messages)
         messages.each { |message| attach(message.to_s.dup, 'text/x.cucumber.log+plain') }
       end
 
-      def attach(file, media_type)
-        super
+      def attach(file, media_type = nil)
+        return super unless File.file?(file)
+
+        content = File.open(file, 'rb', &:read)
```

Comment:
Would it be possible to use this here?

```suggestion
        content = File.read(file, mode: 'rb')
```

https://rubyapi.org/2.5/o/io#method-c-read

Reply:
I think so, yes 👍